manifest.yaml

```diff
@@ -0,0 +1,40 @@
+name: parquetmr
+description: Lyft fork of the Apache Parquet project
```

Comment:
?? I don't think this should go in the Apache project.

Reply:
Sorry... This PR should not have been out... let me close this one.